server/main.ts

```diff
@@ -1,31 +1,71 @@
 import { Meteor } from 'meteor/meteor';
-import { LinksCollection } from '/imports/api/links';
-
-function insertLink(title: string, url: string) {
-  LinksCollection.insert({ title, url, createdAt: new Date() });
-}
+import '/imports/api/api'
+import SimpleSchema from 'simpl-schema'
+import { RadioField } from '/imports/ui/uniforms-react';
+import { Accounts } from 'meteor/accounts-base';
+import { useTracker } from 'meteor/react-meteor-data'
 
 Meteor.startup(() => {
   // If the Links collection is empty, add some data.
-  if (LinksCollection.find().count() === 0) {
-    insertLink(
-      'Do the Tutorial',
-      'https://www.meteor.com/tutorials/react/creating-an-app'
-    );
-
-    insertLink(
-      'Follow the Guide',
-      'http://guide.meteor.com'
-    );
-
-    insertLink(
-      'Read the Docs',
-      'https://docs.meteor.com'
-    );
-
-    insertLink(
-      'Discussions',
-      'https://forums.meteor.com'
-    );
+  //
+});
+
```

Comment:
Den ganzen Rollen Kram könnte man besser in eine Datei im api Ordner ausgliedern.